com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderCompilers.java

```diff
@@ -36,51 +36,56 @@ private ArrayList<ShaderDesc.Language> getPlatformShaderLanguages(ShaderDesc.Sha
             boolean spirvSupported = true;
             boolean hlslSupported = platform == Platform.X86_64Win32;
 
-            switch(platform) {
-                case Arm64MacOS:
-                case X86_64MacOS: {
-                    if (shaderType != ShaderDesc.ShaderType.SHADER_TYPE_COMPUTE) {
-                        shaderLanguages.add(ShaderDesc.Language.LANGUAGE_GLSL_SM330);
-                    }
-                } break;
-                case X86Win32:
-                case X86_64Win32:
-                case X86Linux:
-                case Arm64Linux:
-                case X86_64Linux: {
+            if (platform == Platform.Arm64MacOS ||
+                platform == Platform.X86_64MacOS) {
+                if (shaderType != ShaderDesc.ShaderType.SHADER_TYPE_COMPUTE) {
+                    shaderLanguages.add(ShaderDesc.Language.LANGUAGE_GLSL_SM330);
+                }
+            }
```

Comment:
We should consider an api for the future IPlatform.java to provide these details instead of putting it all in a function here.